github/issues.go

```diff
@@ -56,6 +56,8 @@ type Issue struct {
 	// TextMatches is only populated from search results that request text matches
 	// See: search.go and https://developer.github.com/v3/search/#text-match-metadata
 	TextMatches []TextMatch `json:"text_matches,omitempty"`
+
+	ActiveLockReason *string `json:"active_lock_reason,omitempty"`
```

Comment:
It would be nice to add a descriptive comment here and enumerate the possible values.